libqtile/widget/launchbar.py

```diff
@@ -210,10 +213,11 @@ def draw(self):
                 # display an icon
                 self.drawer.ctx.set_source(self.surfaces[name])
                 self.drawer.ctx.paint()
-            self.drawer.draw(self.offset + xoffset, icon_width + self.padding)
+            self.drawer.draw(offsetx=self.offset + xoffset,
+                             width=icon_width + self.padding)
             xoffset += icon_width + self.padding
 
-    def calculate_width(self):
+    def calculate_length(self):
```

Comment:
This bit of the diff looks a little funny. I understand why you changed the name, but s/width/length is a bit weird. I don't really have any better suggestions, though, so maybe it's ok.